### PR TITLE
Add support for kms sse when uploading a file

### PR DESCRIPTION
**Pull request by eschizoid**

### CODE REVIEW

File: src/main/java/jp/classmethod/aws/gradle/s3/AmazonS3FileUploadTask.java [hunk 1]
```diff
@@ -51,7 +53,7 @@ public void upload() throws IOException {
 		if (file == null) {
 			throw new GradleException("file is not specified");
 		}
-		if (file.isFile() == false) {
+		if (file.isFile()) {
```
Comment (dai0304) [inline src/main/java/jp/classmethod/aws/gradle/s3/AmazonS3FileUploadTask.java]: Why this condition is inverted?

**Comment by eschizoid** [comment]
@dai0304 Fixed

**Comment by eschizoid** [comment]
@dai0304 Can you merge it down? Would like to start using it tomorrow without having to point to my internal maven server :)

**Comment by dai0304** [comment]
ok!

**Pull request merged**